auth-web/src/components/auth/LoginBCSC.vue

```diff
@@ -3,7 +3,7 @@
     <v-card-title>Create a BC Registries Account</v-card-title>
     <v-card-text>
       <p class="mb-7">To access this service, you must securely log in with your BC Services Card, government’s trusted way to access online services. It typically takes five minutes or less to set up your mobile card, and the only information BC Registries can access from your card is your legal name.</p>
-      <a href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/login-with-card" rel="noopener noreferrer"
+      <a href="https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/log-in-with-card" rel="noopener noreferrer"
```

Comment:
I prefer the way you had properties for the URL in the other component. It keeps the template a bit cleaner.

Reply:
Oops, will update this for consistency!

Reply:
Done in next commit!